setup.py

```diff
@@ -38,8 +38,11 @@
 import shutil
```

Comment:
I would appreciate if the old setup would still work (Especially for people who don't want to use the package, or for those making changes in the native modules) - maybe we could split it in two files, the old setup and the new package setup?